torchci/test/jobUtils.test.ts

```diff
@@ -565,4 +569,606 @@ describe("Test various job utils", () => {
       ])
     );
   });
+
+  test("test isDisabledTest", async () => {
```

Comment:
Can I petition you to change the objects in these tests to reuse some base issue and use the spread operator to set specific things you want to change?  The html_url, author_association etc take up a lot of space and make it unclear what is important to the test